types/webgl2/index.d.ts

```diff
@@ -486,21 +510,23 @@ interface WebGL2RenderingContext extends WebGLRenderingContext {
     /* Reading back pixels */
     // WebGL1:
     readPixels(x: number, y: number, width: number, height: number, format: number, type: number,
-        dstData: ArrayBufferView | null): void;
+        dstData: Int8Array | Int16Array | Int32Array | Uint8Array | Uint16Array | Uint32Array | Uint8ClampedArray |
+        Float32Array | Float64Array | DataView | null): void;
     // WebGL2:
     readPixels(x: number, y: number, width: number, height: number, format: number, type: number,
         offset: number): void;
     readPixels(x: number, y: number, width: number, height: number, format: number, type: number,
-        dstData: ArrayBufferView, dstOffset: number): void;
+        dstData: Int8Array | Int16Array | Int32Array | Uint8Array | Uint16Array | Uint32Array | Uint8ClampedArray |
```

Comment:
The overloaded 8 parameter version was introduced in WebGL2, and shouldn't clash with lib.dom.d.ts. If you want to keep the verbose version, please remove `null`. Specifications for WebGL1 and 2 differ here for some reason in how they treat `dstData`... not sure why / if intended.